spec/rubocop/cli/options_spec.rb

```diff
@@ -1470,14 +1476,14 @@ def full_description_of_cop(cop)
       context 'when simple format is specified' do
         it 'outputs with simple format' do
           cli.run(['--format', 'simple', 'example.rb'])
-          expect($stdout.string)
-            .to include(<<~RESULT)
+          expect($stdout.string.include?(<<~RESULT)) # rubocop:todo RSpec/VoidExpect
```

Comment:
[memo] We have temporarily disabled this offense because of incorrect autocorrect.